opal/tools/wrappers/opal_wrapper.c

```diff
@@ -967,7 +967,7 @@ int main(int argc, char *argv[])
                 char* exec_cmd = opal_argv_join(exec_argv, ' ');
```

Comment:
GitHub won't let me on line 946, but if we're going to fix this, let's remove the #if 0'ed code.  It's just asking for a mental lapse that leads to this kind of bug.  I'm also really unhapy about the entire `exec_cmd` vs. `exec_command` naming at all; that's really poor variable naming on our part.

Reply:
Done.